inc/front/cdn.php

```diff
@@ -269,7 +269,7 @@ function rocket_cdn_custom_files( $html ) {
 		 *
 		 * @param array $filetypes Array of file types.
 		 */
-		$filetypes = apply_filters( 'rocket_cdn_custom_filetypes', array( 'mp3', 'ogg', 'mp4', 'm4v', 'avi', 'mov', 'flv', 'swf', 'webm', 'pdf', 'doc', 'docx', 'txt', 'zip', 'tar', 'bz2', 'tgz', 'rar' ) );
+		$filetypes = apply_filters( 'rocket_cdn_custom_filetypes', array( 'mp3', 'ogg', 'mp4', 'm4v', 'avi', 'mov', 'flv', 'swf', 'webm', 'pdf', 'doc', 'docx', 'txt', 'zip', 'tar', 'bz2', 'tgz', 'rar', 'jpg', 'jpeg', 'png', 'gif', 'webp', 'bmp', 'tiff' ) );
```

Comment:
Erf, we also forgot `ico`.
OK, a good place to start is here: https://wpseek.com/function/wp_get_mime_types/
Maybe we can use it? For example:

```php
$filetypes = wp_get_mime_types();
$filetypes = array_keys( $filetypes );
$filetypes = implode( '|', $filetypes );
$filetypes = explode( '|', $filetypes );
```

The only problem is the `'mime_types'` filter, that can be used to add or remove types :/
At least you can look at the mime types it uses, and complete our list.